pkgs/by-name/wa/waybar-lyric/package.nix

```
@@ -0,0 +1,31 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule (finalAttrs: {
+  pname = "waybar-lyric";
+  version = "0.10.0";
+
+  src = fetchFromGitHub {
+    owner = "Nadim147c";
+    repo = "waybar-lyric";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-glY0W/fB57pYrAhIxYSnkJBmqeN6vy0cwrS71CnKeK4=";
```

The hash seems to be wrong. Testing locally I needed:


```suggestion
    hash = "sha256-RqUaak9MH7lE1ez8T+UKm2Eqk0ImePPubfFExNpZqM8=";
```